FLEDGE.md

```diff
@@ -1235,7 +1295,112 @@ The following table summarizes which APIs get original and which get converted b
+passed to `reportWin()` and `reportResult()` is determined by the browser with the following logic:
+
+* If `selectableBuyerAndSellerReportingIds` defined in interest group:
+    * If `selectedBuyerAndSellerReportingId` in bid:
+        * Then `selectedBuyerAndSellerReportingId`, `buyerAndSellerReportingId` (if present in interest group), and `buyerReportingId`
+          (if present in interest group) will all be available to reporting.
+    * Otherwise (i.e. when `selectedBuyerAndSellerReportingId` not in bid):
+        * No reporting IDs. If a buyer wants reporting IDs, they can consider including and selecting an empty string (`''`) member
+          `selectableBuyerAndSellerReportingIds`, for example `selectableBuyerAndSellerReportingIds: [ '', ...` in the interest group
+          and `'selectedBuyerAndSellerReportingId': ''` returned from `generateBid()`.
+* Otherwise (i.e. when `selectableBuyerAndSellerReportingIds` not defined in interest group):
+    * If `buyerAndSellerReportingId` defined in interest group: `buyerAndSellerReportingId` available to reporting.
+    * Otherwise, if `buyerReportingId` defined in interest group: `buyerReportingId` available to reporting.
+    * Otherwise, interest group `name` available to reporting.
+
+Here's a table representation of the above logic: (`selectableBuyerAndSellerReportingIds` abreviated to selectableBASRI,
+`buyerAndSellerReportingId` abreviated to BASRI, `buyerReportingId` abreviated to BRI)
+<table>
+  <thead>
+    <tr>
+      <th colspan=3>When interest group contains:</th>
+      <th colspan=2>then reports get:</th>
+    </tr>
+    <tr>
+      <th>selectableBASRI</th>
+      <th>BASRI</th>
+      <th>BRI</th>
+      <th>reportWin()</th>
+      <th>reportResult()</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>yes and in bid</td>
+      <td>optional</td>
+      <td>optional</td>
+      <td>selectableBASRI, BASRI, BRI</td>
+      <td>selectableBASRI, BASRI</td>
+    </tr>
+    <tr>
+      <td>yes but not in bid</td>
+      <td>optional</td>
+      <td>optional</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>no</td>
+      <td>yes</td>
+      <td>optional</td>
+      <td>BASRI</td>
+      <td>BASRI</td>
+    </tr>
+    <tr>
+      <td>no</td>
+      <td>no</td>
+      <td>yes</td>
+      <td>BRI</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>no</td>
+      <td>no</td>
+      <td>no</td>
+      <td>interest group name</td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+`generateBid()` is passed all reporting IDs in each entry in the interst group's `ads` list, though in cases where `generateBid()`
+is re-run, after the first invocation didn't produced any bids with ads that passed the k-anonymity checks, reporting IDs that
+don't pass the k-anonymity check will not be present in the interest group.
+
+`scoreAd()` is passed the same reporting IDs as `reportResult()`, but without the k-anonymity requirement.  If the seller
```

Comment:
Replace this paragraph altogether with:

```
For bids with a value for `selectedbuyerAndSellerReportingId`, `scoreAd` is passed
the same reporting IDs as would be passed to `reportResult()`
(`selectedBuyerAndSellerReportingId` and `buyerAndSellerReportingId`).
```

We could maybe continue with:
```
To ensure accurate reporting, a bid with a value for
`selectedbuyerAndSellerReportingId` may only win the auction if the value of
`selectedbuyerAndSellerReportingId` is jointly k-anonymous along with
`buyerAndSellerReportingId`, `buyerReportingId`, the interest group owner,
bidding script URL, and render URL, so that `reportResult()` will always receive
the value of `selectedbuyerAndSellerReportingId` for bids that provided a value.
```